vaccheck.js

```diff
@@ -15,6 +15,13 @@ javascript:(function(){
                     span.style.fontWeight = 'bold';
                     span.innerHTML = data.players[0].NumberOfVACBans + ' VAC bans ' + data.players[0].DaysSinceLastBan + ' days ago.';
                     friend.parentNode.parentNode.querySelector('.friendSmallText').appendChild(span);
+                }else{
+                    var span = document.createElement('span');
+                    span.style.display = 'block';
+                    span.style.color = 'rgb(43, 203, 64)';
+                    span.style.fontWeight = 'bold';
+                    span.innerHTML = 'No VAC Bans for this player.';
+                    friend.parentNode.parentNode.querySelector('.friendSmallText').appendChild(span);
```

Comment:
I'd clean this up a tad first. Create the span outside of the if statement, and just set the color and inner text in the conditional. There's some repeated code and I'm kind of a stickler for keeping code clean :smile: Feel free to make the change and I'll gladly merge it in!